components/schema-migrator/postgres/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM postgres:11
```

Comment:
why do you create another Docker image? you can mount the script to docker-entrypoint-initdb.d when running 
 ordinary postgres image.